app/src/main/java/com/example/android/codeforces/Activities/WebViewActivity.java

```diff
@@ -0,0 +1,27 @@
+package com.example.android.codeforces.Activities;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.webkit.WebView;
+import com.example.android.codeforces.R;
+import static com.example.android.codeforces.Constants.contestUrlKey;
+
+public class WebViewActivity extends AppCompatActivity {
+
+    private WebView webView;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_webview);
+
+        String url = "";
```

Comment:
Why here url like this
 if the intent is null then it may get the null pointer exception and it may crash

Reply:
The base url is constant and every contest has id which is not optional. That's why I didn't put any check.

Reply:
Yeah, I got it